ZEngine/src/VulkanInstance.cpp

```diff
@@ -125,27 +130,36 @@ namespace ZEngine::Hardwares
 
         for (const VkPhysicalDevice& physical_device : physical_device_collection)
         {
+            std::unordered_set<std::string_view> device_enabled_layer_name_collection;
+            std::unordered_set<std::string_view> device_extension_layer_name_collection = {VK_KHR_SWAPCHAIN_EXTENSION_NAME};
+
             for (LayerProperty& layer : layer_properties)
```

Comment:
This also doesn't work on my machine. Same as for the instance creation, enabling all layers and extensions available on the physical device is not a good idea. Decide in advance, which layers and extensions you need (if any), check if they are available here and then only enable these, or error out.

Reply:
Completed removed this file and its implementation.
Thanks a lot !